_sass/_distill.scss

```
@@ -48,8 +49,102 @@ d-article {
-    border-left: 2px solid var(--global-theme-color) !important;
+    background: var(--global-bg-color);
+    border-left: 5px solid var(--global-theme-color);
+    margin: 1.5em 0;
+    padding: 1em;
+    font-size: 1.2rem;
+    font-style: italic;
+
+    p {
+      margin-bottom: 0;
+    }
+    // Italics
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)